docs/deploy/configurations.md

```diff
@@ -127,6 +127,16 @@ Once disabled:
 * you can no longer create Ingresses with the `alb.ingress.kubernetes.io/group.name` annotation.
 * you can no longer alter the value of an `alb.ingress.kubernetes.io/group.name` annotation on an existing Ingress.
 
+### sync-period
+`--sync-period` defines a fix interval for the controller to auto reconcile, default to 1hr. Please be mindful that frequent reconciliations may incur unnecessary AWS API usage.
```

Comment:
let's also change the default sync-period to 10 hour instead 1 hour, which matches controller-runtime's default value: https://github.com/kubernetes-sigs/controller-runtime/blob/56a973c46efe463ef7b5fbecd8495e8378b50d20/pkg/cache/cache.go#L43

need to update documentation here as well: https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/helm/aws-load-balancer-controller/values.yaml#L218